tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -1042,6 +1042,75 @@ describe('migrate-converged-pkg generator', () => {
     });
   });
 
+  describe(`just-scripts config setup`, () => {
+    it(`should update just.config.ts`, async () => {
+      function getJustConfig(projectConfig: ReadProjectConfiguration) {
+        return tree.read(`${projectConfig.root}/just.config.ts`)?.toString('utf-8');
+      }
+      const projectConfig = readProjectConfiguration(tree, options.name);
+      let justConfig = getJustConfig(projectConfig);
+
+      expect(justConfig).toMatchInlineSnapshot(`
+        "import { preset } from '@fluentui/scripts-tasks';
+
+        preset();"
+      `);
+
+      await generator(tree, options);
+
+      justConfig = getJustConfig(projectConfig);
+
+      expect(justConfig).toMatchInlineSnapshot(`
+        "import { preset, task } from '@fluentui/scripts-tasks';
+
+        preset();
+
+        task('build', 'build:react-components').cached!();"
+      `);
+    });
+  });
+
+  describe(`swcrc config setup`, () => {
+    it(`should create an swcrc config file`, async () => {
+      function getSwcConfig(projectConfig: ReadProjectConfiguration) {
+        return readJson(tree, `${projectConfig.root}/.swcrc`);
+      }
+      const projectConfig = readProjectConfiguration(tree, options.name);
+
+      await generator(tree, options);
+
+      expect(tree.exists(`${projectConfig.root}/.swcrc`)).toBeTruthy();
```

Comment:
should we add v-build to swrc files as codeowners?

Reply:
yes pls